src/Components/Server/src/ComponentHub.cs

```diff
@@ -240,7 +241,7 @@ public async ValueTask<bool> ReceiveJSDataChunk(long streamId, long chunkId, byt
             return await circuitHost.ReceiveJSDataChunk(streamId, chunkId, chunk, error);
         }
 
-        public async ValueTask DispatchBrowserEvent(string eventDescriptor, string eventArgs)
+        public async ValueTask DispatchBrowserEvent(JsonElement eventDescriptor, JsonElement eventArgs)
```

Comment:
Careful with changing public hub methods. Old clients will still pass strings which would break. Maybe old clients aren't a concern for blazor apps though 🤷‍♀️